openstack/datadog_checks/openstack/openstack.py

```diff
@@ -1015,32 +1024,26 @@ def _is_valid_metric(label):
 
         project_name = project.get('name')
 
```

Comment:
let's create a `project_id` variable and reuse it below like you did with `project_name`